org.eclipse.transformer/src/main/java/org/eclipse/transformer/Transformer.java

```diff
@@ -283,7 +283,11 @@ public static Options build(OptionSettings[] settings) {
 			!OptionSettings.IS_REQUIRED, OptionSettings.NO_GROUP),
 
 		DRYRUN("d", "dryrun", "Dry run", !OptionSettings.HAS_ARG, !OptionSettings.HAS_ARGS, !OptionSettings.IS_REQUIRED,
-			OptionSettings.NO_GROUP);
+			OptionSettings.NO_GROUP),
+                
+                RULES_PER_CLASS_CONSTANT("tcd", "per-class-constant", "Transformation per class constant string replacements",
```

Comment:
Remember to add this option to the Jakarta transformer and to the rules documentation.

Reply:
Thanks, done (added also to Maven plugin).